### PR TITLE
Editors: valueChanged handler should receive correct event parameter

### PR DESCRIPTION
**Pull request by ksercs**

### CODE REVIEW

File: js/ui/color_box/color_box.js [hunk 1]
```diff
@@ -192,9 +192,10 @@ const ColorBox = DropDownEditor.inherit({
             applyValueMode: that.option('applyValueMode'),
             focusStateEnabled: that.option('focusStateEnabled'),
             stylingMode: this.option('stylingMode'),
-            onEnterKeyPressed: function() {
+            onEnterKeyPressed: function(args) {
```
Comment (DokaRus) [inline js/ui/color_box/color_box.js]:
```suggestion
            onEnterKeyPressed: function({ event, value }) {
```

File: js/ui/color_box/color_box.js [hunk 2]
```diff
@@ -192,9 +192,10 @@ const ColorBox = DropDownEditor.inherit({
...
                 that._colorViewEnterKeyPressed = true;
                 if(that._colorView.option('value') !== that.option('value')) {
+                    that._saveValueChangeEvent(args.event);
```
Comment (DokaRus) [inline js/ui/color_box/color_box.js]:
```suggestion
                    that._saveValueChangeEvent(event);
```

File: js/ui/color_box/color_box.js [hunk 3]
```diff
@@ -211,12 +212,13 @@ const ColorBox = DropDownEditor.inherit({
                     return;
                 }
 
+                args.event && that._saveValueChangeEvent(args.event);
```
Comment (DokaRus) [inline js/ui/color_box/color_box.js]:
```suggestion
                if(event) {
                  that._saveValueChangeEvent(event);
                }
```

File: js/ui/color_box/color_box.js [hunk 4]
```diff
@@ -211,12 +212,13 @@ const ColorBox = DropDownEditor.inherit({
...
                 that._applyNewColor(args.value);
```
Comment (DokaRus) [inline js/ui/color_box/color_box.js]:
```suggestion
                that._applyNewColor(value);
```

File: js/ui/color_box/color_view.js [hunk 5]
```diff
@@ -374,9 +382,10 @@ const ColorView = Editor.inherit({
             boundOffset: (function() {
                 return -this._paletteHandleHeight / 2;
             }).bind(this),
-            onDragMove: (function() {
+            onDragMove: (function(args) {
```
Comment (DokaRus) [inline js/ui/color_box/color_view.js]:
```suggestion
            onDragMove: ({ event }) => {
```

and remove `.bind(this)` below

File: js/ui/color_box/color_view.js [hunk 6]
```diff
@@ -442,8 +451,9 @@ const ColorView = Editor.inherit({
             boundary: this._$hueScaleWrapper,
             allowMoveByClick: true,
             dragDirection: 'vertical',
-            onDragMove: (function() {
+            onDragMove: (function(args) {
```
Comment (DokaRus) [inline js/ui/color_box/color_view.js]:
```suggestion
            onDragMove: ({ event }) => {
```

File: js/ui/color_box/color_view.js [hunk 7]
```diff
@@ -703,11 +714,11 @@ const ColorView = Editor.inherit({
             boundary: $parent,
             allowMoveByClick: true,
             dragDirection: 'horizontal',
-            onDragMove: (function() {
+            onDragMove: (function(args) {
```
Comment (DokaRus) [inline js/ui/color_box/color_view.js]:
```suggestion
            onDragMove: ({ event }) => {
```

File: js/ui/date_box/ui.date_box.strategy.calendar.js [hunk 8]
```diff
@@ -118,7 +118,7 @@ const CalendarStrategy = DateBoxStrategy.inherit({
                     onInitialized: function(e) {
                         e.component.registerKeyHandler('escape', this._escapeHandler.bind(this));
                     }.bind(this),
-                    onClick: (function() { this._widget._toTodayView(); }).bind(this),
+                    onClick: (function(args) { this._widget._toTodayView(args); }).bind(this),
```
Comment (DokaRus) [inline js/ui/date_box/ui.date_box.strategy.calendar.js]:
```suggestion
                    onClick: (args) => { this._widget._toTodayView(args); },
```

**Review by DokaRus — DISMISSED** [review]

**Review by alexander-kotov-dx — DISMISSED** [review]

**Review by DokaRus — APPROVED** [review]

**Review by alexander-kotov-dx — APPROVED** [review]

**Pull request merged**